File: Views/src/components/services-provision/bench/dto/bench-to-Cash-Box-Dto.ts
import { ServiceBenchDto } from "./service-bench-dto";

export class BenchToCashBoxDto {
  id:number;
  technician:string;
  priceService:number;
  problemByTechnician:string;
  technicalSolutionApplied:string;
  // cantBeSolved:string;
  status:string;
  solved:boolean;
  // hardware:boolean;
  serviceBenchId:number;
  serviceBench:ServiceBenchDto;
}

File: Application/Dto/ServiceBudgetBench/BenchToCashBoxDto.cs

namespace Application.Dto.ServiceBudgetBench

{
    public class BenchToCashBoxDto
    {
        public int Id { get; set; }
        public string Technician { get; set; }
        public decimal PriceService { get; set; }
        public string ProblemByTechnician { get; set; }
        public string TechnicalSolutionApplied { get; set; }
        // public string CantBeSolved { get; set; }
        public string Status { get; set; }
        public bool Solved { get; set; }
        // public bool Hardware { get; set; }
        public int ServiceBenchId { get; set; }
        public ServiceBenchDto ServiceBench { get; set; }
    }
}
